.github/workflows/cross-version-tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           fi
 
           if [ "${{ github.event_name }}" = "pull_request" ]; then
-            REF_VERSIONS_YAML="https://raw.githubusercontent.com/mlflow/mlflow/master/ml-package-versions.yml"
+            REF_VERSIONS_YAML="https://raw.githubusercontent.com/mlflow/mlflow/master/mlflow/ml-package-versions.yml"
```

Comment:
Is this a valid file URL?

Reply:
I move ml-package-versions.yml into `mlflow` sub-directory. So that when install package, it will be installed inside the `.../site-package/mlflow/ml-package-versions.yml`
otherwise it will be installed in `.../site-package/ml-package-versions.yml`

Reply:
ah I missed that.